.github/workflows/pythonpackage.yml

```diff
@@ -50,11 +51,17 @@ jobs:
           # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
           flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
 
+      - name: Test third_party 
+        run: |
+          pip install pytest
+          pytest third_party/quaff
```

Comment:
you can probably remove this in a following PR when we have tests that exercise the quaff code path. We don't usually run tests for external packages